app/src/js/obj-validation.js

```diff
@@ -86,13 +86,16 @@ Bolt.validation = (function () {
      */
     function showAlertbox(id, field, msg) {
         var alertbox;
+        var fieldname = $('label[for="' + field.id + '"]').contents().first().text().trim() || field.name;
+
+        fieldname = fieldname.replace(/(.*):+$/g, "$1" );
```

Comment:
It takes the labels, as set in `label: "Title:"`. I tend to include a colon in there, other people might not.